apps/web/src/lib/site-initializer/index.ts

```diff
@@ -51,12 +52,21 @@ const removeInitializer = () => {
 
 const init = async (store) => {
     /* Init SpaceONE Console */
-    await initConfig();
-    await initApiClient(store, config);
-    const domainName = await initDomain(store, config);
+    try {
+        await initConfig();
+        await initApiClient(store, config);
+        await initDomain(store, config);
+    } catch (e) {
+        initRouter();
+        throw new Error('Site initialization failed: No matched domain');
+    }
 
-    if (domainName) {
-        initModeSetting();
+    const domainId = store.state.domain.domainId;
+    initModeSetting();
+    initRouter(domainId);
+    const userId = await initUserAndAuth(store, config);
```

Comment:
issue: What if initiating user process failed? I think this process also must be in the try block above.